ergo/platforms/metaculus/metaculus.py

```diff
@@ -67,14 +69,21 @@ class Metaculus:
         "interested": "upvoted_by",
     }
 
-    def __init__(self, username: str, password: str, api_domain: str = "www"):
-        self.user_id = None
+    def __init__(
+        self,
+        api_domain: Optional[str] = "www",
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+    ):
+        if username and password:
```

Comment:
should be `or` I think

Reply:
Yeah I wasn't really sure which it should be, I'll just make it `or`, not sure it matters much.